### PR TITLE
fix(VCarousel): incorrect transition direction when using touch

### LINKED ISSUE #11280: [Bug Report] v-carousel slides in the wrong direction on first & last image with touch controls
### Environment
**Vuetify Version:** 2.2.26
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.129, Google Chrome, Mozilla Firefox
**OS:** Windows 10, Android

### Steps to reproduce
This example is taken from the Vuetify documentation https://vuetifyjs.com/en/components/carousels/ so you can go there and test as well. Use either a phone or Chrome's touch simulation in the Devtool and slide to the right on the first image in the carousel. Compare it to how it behaves when the arrows are used. 

### Expected Behavior
I expect the touch to behave like the arrow keys. To transition from the last image to the first as if they were beside each other. 

### Actual Behavior
Now it instead is really yanky and goes in the other direction as if it was going back to the start of the slideshow instead of seamlessly restarting

### Reproduction Link
<a href="https://codepen.io/hugogranstrom/pen/jObLomO" target="_blank">https://codepen.io/hugogranstrom/pen/jObLomO</a>

### PR DESCRIPTION
**Pull request by nekosaur**
## Description
using touch would result in incorrect transition when looping around

## Motivation and Context
closes #11280

## Markup:

<details>

```vue
<template>
  <div>
    <v-switch
      v-model="$vuetify.rtl"
      label="RTL"
    />
    <v-btn @click="foo = foo - 1 < 0 ? slides.length - 1 : foo - 1">prev</v-btn>
    <v-btn @click="foo = foo + 1 >= slides.length ? 0 : foo + 1">next</v-btn>
    <v-carousel
      v-model="foo"
      height="400"
      hide-delimiter-background
      show-arrows-on-hover
    >
      <v-carousel-item
        v-for="(slide, i) in slides"
        :key="i"
      >
        <v-sheet
          :color="colors[i]"
          height="100%"
        >
          <v-row
            class="fill-height"
            align="center"
            justify="center"
          >
            <div class="display-3">{{ slide }} Slide</div>
          </v-row>
        </v-sheet>
      </v-carousel-item>
    </v-carousel>

    <v-carousel
      height="400"
      hide-delimiter-background
      show-arrows-on-hover
      reverse
    >
      <v-carousel-item
        v-for="(slide, i) in slides"
        :key="i"
        :value="`value_${i}`"
      >
        <v-sheet
          :color="colors[i]"
          height="100%"
        >
          <v-row
            class="fill-height"
            align="center"
            justify="center"
          >
            <div class="display-3">{{ slide }} Slide</div>
          </v-row>
        </v-sheet>
      </v-carousel-item>
    </v-carousel>

    <v-carousel
      height="400"
      hide-delimiter-background
      show-arrows-on-hover
      vertical
    >
      <v-carousel-item
        v-for="(slide, i) in slides"
        :key="i"
        :value="`value_${i}`"
      >
        <v-sheet
          :color="colors[i]"
          height="100%"
        >
          <v-row
            class="fill-height"
            align="center"
            justify="center"
          >
            <div class="display-3">{{ slide }} Slide</div>
          </v-row>
        </v-sheet>
      </v-carousel-item>
    </v-carousel>
  </div>
</template>

<script>
  export default {
    data: () => ({
      foo: 0,
      colors: [
        'indigo',
        'warning',
        'pink darken-2',
        'red lighten-1',
        'deep-purple accent-4',
      ],
      slides: [
        'First',
        'Second',
        'Third',
        'Fourth',
        'Fifth',
      ],
    }),
  }
</script>

```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VWindow/VWindow.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default BaseItemGroup.extend({
     },
     reverse: {
```
Comment (johnleider) [inline packages/vuetify/src/components/VWindow/VWindow.ts:47]: Any reason against doing `reverse: Boolean`?

**Review by johnleider — CHANGES_REQUESTED** [review]

File: packages/vuetify/src/components/VWindow/VWindow.ts (hunk 1 above)
Comment (nekosaur) [reply packages/vuetify/src/components/VWindow/VWindow.ts:47]: no

File: packages/vuetify/src/components/VWindow/VWindow.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ export default BaseItemGroup.extend({
...
       type: Boolean,
-      default: undefined,
+      default: false,
```
Comment (johnleider) [inline packages/vuetify/src/components/VWindow/VWindow.ts:49]: Booleans are false by default.

File: packages/vuetify/src/components/VWindow/VWindow.ts [hunk 3]
```diff
@@ -149,10 +150,7 @@ export default BaseItemGroup.extend({
             'aria-label': this.$vuetify.lang.t(`$vuetify.carousel.${direction}`),
           },
           on: {
-            click: () => {
-              this.changedByDelimiters = true
-              fn()
-            },
+            click,
```
Comment (johnleider) [inline packages/vuetify/src/components/VWindow/VWindow.ts:153]: could be on 1 line on 152

**Review by johnleider — CHANGES_REQUESTED** [review]

**Review by MajesticPotatoe — APPROVED** [review]
LG!

**Pull request merged**